### PR TITLE
Feature/dx 1001 correction view operations

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: _includes/payment-resource.md [hunk 1]
```diff
@@ -196,10 +238,24 @@ for the given operation.
...
 | `redirect-sale`        | Contains the redirect-URI that redirects the consumer to a Swedbank Pay hosted payment page prior to creating a sales transaction.                                        |
 | `view-sales`           | Contains the URI of the JavaScript used to create a Seamless View iframe directly for the `sale` transaction without redirecting the consumer to a separate payment page. |
-| `view-payment`         | Contains the URI of the JavaScript used to create a Hosted View iframe directly without redirecting the consumer to separate payment page.                                |
+| `view-payment`         | Contains the URI of the JavaScript used to create a Seamless View iframe directly without redirecting the consumer to separate payment page.                                |
+
 
-{% else %}
-{% if show_status_operations %}
+
+{% when "trustly" %}
```
Comment (payex-dev) [inline _includes/payment-resource.md:245]: Codacy found an issue: [[no-consecutive-blank-lines] Remove 2 lines before node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5718884)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5718884&bid=18712408)

File: _includes/payment-resource.md [hunk 2]
```diff
@@ -107,7 +103,36 @@ Content-Type: application/json
             "method": "GET",
             "href": "{{ page.front_end_url }}/{{ api_resource }}/core/scripts/client.js?token={{ page.payment_token }}",
             "rel": "view-payment"
-        }{% else %},
+            "contentType:": "application/javascript"
+        },
+        {% when "trustly" %}
```
Comment (asbjornu) [inline _includes/payment-resource.md:108]: Please ensure that this `when` doesn't introduce extra line-feeds. If it does, try adding `-` to the liquid tags to remove whitespace.

```suggestion
        {%- when "trustly" -%}
```

File: _includes/payment-resource.md [hunk 3]
```diff
@@ -196,10 +238,24 @@ for the given operation.
 | `create-sale`          | Creates a `sales` transaction without redirection to a payment page.                                                                                                      |
 | `redirect-sale`        | Contains the redirect-URI that redirects the consumer to a Swedbank Pay hosted payment page prior to creating a sales transaction.                                        |
 | `view-sales`           | Contains the URI of the JavaScript used to create a Seamless View iframe directly for the `sale` transaction without redirecting the consumer to a separate payment page. |
-| `view-payment`         | Contains the URI of the JavaScript used to create a Hosted View iframe directly without redirecting the consumer to separate payment page.                                |
+| `view-payment`         | Contains the URI of the JavaScript used to create a Seamless View iframe directly without redirecting the consumer to separate payment page.                                |
+
 
-{% else %}
-{% if show_status_operations %}
+
```
Comment (asbjornu) [inline _includes/payment-resource.md:244]:
```suggestion
```

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5718884&bid=18712408)

File: _includes/payment-resource.md [hunk 4]
```diff
@@ -196,10 +238,23 @@ for the given operation.
 | `create-sale`          | Creates a `sales` transaction without redirection to a payment page.                                                                                                      |
 | `redirect-sale`        | Contains the redirect-URI that redirects the consumer to a Swedbank Pay hosted payment page prior to creating a sales transaction.                                        |
 | `view-sales`           | Contains the URI of the JavaScript used to create a Seamless View iframe directly for the `sale` transaction without redirecting the consumer to a separate payment page. |
-| `view-payment`         | Contains the URI of the JavaScript used to create a Hosted View iframe directly without redirecting the consumer to separate payment page.                                |
+| `view-payment`         | Contains the URI of the JavaScript used to create a Seamless View iframe directly without redirecting the consumer to separate payment page.                                |
 
-{% else %}
-{% if show_status_operations %}
+
+{% when "trustly" %}
```
Comment (payex-dev) [inline _includes/payment-resource.md:244]: Codacy found an issue: [[no-consecutive-blank-lines] Remove 2 lines before node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5718884)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5718884&bid=18712408)

**Pull request merged**